packages/relay/src/lib/eth.ts

```diff
@@ -1537,7 +1544,8 @@ export class EthImpl implements Eth {
         .labels(EthImpl.ethSendRawTransaction, transaction.substring(0, constants.FUNCTION_SELECTOR_CHAR_LENGTH))
         .inc();
 
-    const parsedTx = await this.parseRawTxAndPrecheck(transaction, requestIdPrefix);
+    const networkGasPriceInWeiBars = await this.gasPrice(requestIdPrefix);
```

Comment:
Q: Is there any dead code that needs to be removed

Reply:
I also looked into this and for now there doesn't appear to be any dead code. Everything seems to be in order for now. 🤞🏼